src/bundle/Security/TwoFactor/Provider/DefaultTwoFactorFormRenderer.php

```diff
@@ -20,15 +20,18 @@ class DefaultTwoFactorFormRenderer implements TwoFactorFormRendererInterface
      */
     private $template;
 
-    public function __construct(Environment $twigRenderer, string $template)
+    private array $context;
```

Comment:
1) Please annotate this with PHPDoc type `array`
2)  `$context` doesn't make clear what it is. Let's name this `$templateVars`. Please also update the constructur argument.

Reply:
Please use PHPDoc instead of PHP 7.4 type hints. The library has to support PHP versions 7.2 and 7.3, which don't have that language feature.